src/main/kotlin/racingcar/util/CarsFactory.kt

```diff
@@ -0,0 +1,12 @@
+package racingcar.util
+
+import racingcar.domain.Car
+import racingcar.domain.Cars
+
+object CarsFactory {
+    fun createCars(numberGenerator: NumberGenerator, carCount: Int): Cars = Cars(
+        Array(carCount) {
```

Comment:
```suggestion
        List(carCount) {
```
가급적이면 Array 대신 List 사용을 권장합니다.
Array는 외부에서도 데이터 조작이 가능(Mutable)하기에 지양하며, List를 사용하면 외부에서 데이터 조작이 불가능(Immutable)하기에 권장합니다. 😄